Added extra documentation for Windows Audit Policies in Syscheck whodata (#5676)
wazuh/wazuh-documentation: source/user-manual/capabilities/auditing-whodata/who-windows.rst

How it works
^^^^^^^^^^^^

> The who-data monitoring functionality uses the Microsoft Windows auditing subsystem to get the related information about who made modifications in a monitored directory. These changes produce audit events that are processed by *syscheck* and reported to the manager. This feature is only compatible with systems greater than Windows Vista.

Author: Done

> For the correct operation of the who-data feature, it is necessary to have configured some Local Audit Policies as well as the SACLs of each monitored directory. Wazuh automatically performs this task when the tag ``whodata="yes"`` is declared within the ``directories`` statement in the ``ossec.conf`` file:

Author: Done

> For the correct operation of this auditing subsystem, it is necessary to have configured some Local Audit Policies and the SACLs of each configured directory from which to receive these events. Wazuh automatically performs this task when the tag ``whodata="yes"`` is declared within the ``directories`` statement in the ``ossec.conf`` file:

Reviewer: ```suggestion
For the correct operation of the who-data feature, it is necessary to have configured some Local Audit Policies as well as the SACLs of each monitored directory. Wazuh automatically performs this task when the tag ``whodata="yes"`` is declared within the ``directories`` statement in the ``ossec.conf`` file:
```

Author: Done

.. code-block:: xml

    <syscheck>
      <directories check_all="yes" whodata="yes">C:\Windows\System32\drivers\etc</directories>
    </syscheck>

> However, other processes could modify some of these configurations, preventing syscheck from receiving the necessary information. In this case the directories configured in whodata mode will be reconfigured to realtime mode, so that syscheck will continue to generate alerts. Wazuh detects these changes in two different ways:

Reviewer: ```suggestion
Even though FIM configures the needed Local Audit Policies and SASLs when launched. Other services may change this configuration preventing who-data from receiving the monitored events. In this case, FIM detects this configuration change and switches all the directories configured with who-data to real-time mode. The two available mechanisms to detect these configuration changes are:
```

Author: Done

> Even though FIM configures the needed Local Audit Policies and SASLs when launched, other services may change this configuration preventing who-data from receiving the monitored events. In this case, FIM detects this configuration change and switches all the directories configured with who-data to real-time mode. The two available mechanisms to detect these configuration changes are:

Author: Done

> #. Windows generates specific events (ID 4719) when one of the Audit Policies is modified (Success removed).

Author: Done

> #. Periodically, Wazuh checks that the Audit Policies and the SACLs are configured as expected. It is possible to modify this interval, see :ref:`the configuration section of windows_audit_interval<reference_ossec_syscheck_windows_audit_interval>`.

Reviewer: ```suggestion
#. Periodically, Wazuh checks that the Audit Policies and the SACLs are configured as expected. It is possible to modify the frequency of this verification with :ref:`windows_audit_interval<reference_ossec_syscheck_windows_audit_interval>`.
```

Author: Done

> #. Periodically, Wazuh checks that the Audit Policies and the SACLs are configured as expected. It is possible to modify the frequency of this verification with :ref:`windows_audit_interval<reference_ossec_syscheck_windows_audit_interval>`.

Author: Done
